### ISSUE TITLE
install   installs "twice" for some reason

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#787]
```
$> datalad install ///                                
2016-09-09 14:53:05,521 [INFO   ] Installing <Dataset path=/tmp/datasets.datalad.org> from http://datasets.datalad.org/ (install.py:371)
1 installed item is available at
<Dataset path=/tmp/datasets.datalad.org>
(venv-tests) 2 13997 [1].....................................:Fri 09 Sep 2016 02:53:05 PM EDT:.
hopa:/tmp
$> cd datasets.datalad.org 
crcns/  indi/  kaggle/  labs/  neurovault/  nidm/  openfmri/
(venv-tests) 2 13998 [1].....................................:Fri 09 Sep 2016 02:53:08 PM EDT:.
(git)hopa:/tmp/datasets.datalad.org[master]
$> datalad install nidm 
2016-09-09 14:53:13,599 [WARNING] received 'None' as 'source'. (install.py:55)
2016-09-09 14:53:13,599 [INFO   ] Installing /tmp/datasets.datalad.org/nidm (install.py:371)
2016-09-09 14:53:13,620 [INFO   ] Installing <Dataset path=/tmp/datasets.datalad.org/nidm> from http://datasets.datalad.org/nidm/.git (install.py:371)
1 installed item is available at
<Dataset path=/tmp/datasets.datalad.org/nidm>
(venv-tests) 2 13999 [1].....................................:Fri 09 Sep 2016 02:53:13 PM EDT:.
(git)hopa:/tmp/datasets.datalad.org[master]
$> datalad install nidm/fbirn_phaseIII 
2016-09-09 14:53:15,787 [WARNING] received 'None' as 'source'. (install.py:55)
2016-09-09 14:53:15,788 [INFO   ] Installing /tmp/datasets.datalad.org/nidm/fbirn_phaseIII (install.py:371)
2016-09-09 14:53:15,849 [INFO   ] Installing <Dataset path=/tmp/datasets.datalad.org/nidm/fbirn_phaseIII> from http://datasets.datalad.org/nidm/fbirn_phaseIII/.git (install.py:371)
2016-09-09 14:53:18,103 [INFO   ] Installing <Dataset path=/tmp/datasets.datalad.org/nidm/fbirn_phaseIII> from http://datasets.datalad.org/nidm/./fbirn_phaseIII/.git (install.py:371)
1 installed item is available at
<Dataset path=/tmp/datasets.datalad.org/nidm/fbirn_phaseIII>
(venv-tests) 2 14000 [1].....................................:Fri 09 Sep 2016 02:53:22 PM EDT:.
(git)hopa:/tmp/datasets.datalad.org[master]
$> cat nidm/.gitmodules 
[submodule "fbirn_phaseIII"]
	path = fbirn_phaseIII
	url = ./fbirn_phaseIII
````

**Comment by yarikoptic** [comment]
@bpoldrack : now (pr #613 0.2.3-1029-g2889d29) it is reporting install twice (but differently), reports initializing annex twice (not sure how bad or good that is), and then reports some 'saving' (although none is due AFAIK):

```
$> datalad install nidm/fbirn_phaseIII 
2016-09-19 21:03:26,213 [INFO   ] Installing /tmp/datasets.datalad.org/nidm/fbirn_phaseIII (install.py:434)
2016-09-19 21:03:26,213 [INFO   ] Install subdataset at: /tmp/datasets.datalad.org/nidm/fbirn_phaseIII (install.py:452)
2016-09-19 21:03:26,228 [INFO   ] Installing /tmp/datasets.datalad.org/nidm/fbirn_phaseIII from http://datasets.datalad.org/nidm/fbirn_phaseIII/.git (install.py:434)
2016-09-19 21:03:26,228 [INFO   ] Install dataset at: /tmp/datasets.datalad.org/nidm/fbirn_phaseIII (install.py:492)
2016-09-19 21:03:33,007 [INFO   ] Initializing annex repo at /tmp/datasets.datalad.org/nidm/fbirn_phaseIII (install.py:544)
2016-09-19 21:03:33,038 [INFO   ] Initializing annex repo at /tmp/datasets.datalad.org/nidm/fbirn_phaseIII (install.py:544)
2016-09-19 21:03:33,040 [INFO   ] Saving changes to <Dataset path=/tmp/datasets.datalad.org/nidm> (install.py:592)
1 installed item is available at
<Dataset path=/tmp/datasets.datalad.org/nidm/fbirn_phaseIII>
```

**Comment by bpoldrack** [comment]
Thanks. Looking into it ...

**Comment by bpoldrack** [comment]
Update: Should be fixed by now in #613 .
